index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>Portfolio-Mobile-First</title>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width,initial-scale=1">
+        <link rel="stylesheet" href="style.css">
+    </head>
+    <body>
+        <div class="wrapper">
+            <header>
+                <div class="logo">Welcome</div>
```

Comment:
- [OPTIONAL] Every logo on the website is a link to the home page of that website. Kindly change the generic `div` tag to `<a>` tag. But it is marked optional because the previous reviewer did not mention it. It is highly recommended you give it attention.